scripts/query-tracing.bcc

```diff
@@ -0,0 +1,260 @@
+#! /usr/bin/python3
+# @lint-avoid-python-3-compatibility-imports
+#
+# query_tracing - Show VAST query statistics
+
+from __future__ import print_function
+from bcc import BPF, USDT
+import ctypes
+from ctypes import c_int, c_ulonglong, c_byte, c_ushort
+from sys import argv
+
+
+def usage():
+    print("USAGE: %s /path/to/libvast.so" % argv[0])
```

Comment:
Should probably be piped to a `query_timings.tsv` file?

Reply:
Probably, but that's up to the caller and no concern to the script.